src/ocrd_network/processing_server.py

```diff
@@ -79,14 +79,14 @@ class ProcessingServer(FastAPI):
     """
 
     def __init__(self, config_path: str, host: str, port: int) -> None:
-        initLogging()
         self.title = "OCR-D Processing Server"
         super().__init__(
             title=self.title,
             on_startup=[self.on_startup],
             on_shutdown=[self.on_shutdown],
             description="OCR-D Processing Server"
         )
+        initLogging()
```

Comment:
Oh, any particular reason for calling our init before the superclass? (I cannot find anything in `fastapi.Application` or its superclass `starlette.Starlette` that would effectivate `logging.basicConfig`, and thus interfere with our setup, but maybe I am missing something...)

Anyways, just curious.

Reply:
> Oh, any particular reason for calling our init before the superclass?

Do you mean calling the init after the superclass since that is the change? I moved that to ensure that the superclass does not currently and would not potentially affect the logging outcome in the future (with new releases).

Reply:
Worth mentioning is that this PR does not completely fix the logging of the Processing Server. The log file still contains fewer logs than the ones appearing on the console.

Reply:
> > Oh, any particular reason for calling our init before the superclass?
> 
> Do you mean calling the init after the superclass since that is the change? 

Yes, I did (just forgot the word `not`), sry

> I moved that to ensure that the superclass does not currently and would not potentially affect the logging outcome in the future (with new releases).

Hmm. But is that (FastAPI interfering with our logging setup) not exactly what this change risks bringing about? It seems you assume that our `initLogging` can safely undo whatever other logging setup has been done at that point.

Reply:
> Worth mentioning is that this PR does not completely fix the logging of the Processing Server. The log file still contains fewer logs than the ones appearing on the console.

Could you try tracing all the `.handlers` objects of the relevant loggers?

Reply:
> Hmm. But is that (FastAPI interfering with our logging setup) not exactly what this change risks bringing about?

Now I see what you mean. 

> It seems you assume that our initLogging can safely undo whatever other logging setup has been done at that point.

I assume that we can stay flexible by still being able to overwrite FastAPI loggers with the `ocrd_logging.conf` as well. But that of course could then break FastAPI loggers if not implemented properly. 

I am not sure which is preferable/safe in our case: pre- or post- superclass `initLogging` call.

Reply:
> Could you try tracing all the .handlers objects of the relevant loggers?

Yes, I will try that.

Reply:
> I assume that we can stay flexible by still being able to overwrite FastAPI loggers with the `ocrd_logging.conf` as well. But that of course could then break FastAPI loggers if not implemented properly.
> I am not sure which is preferable/safe in our case: pre- or post- superclass `initLogging` call.

Yes. Or conversely, if they get to init after us, our preferences could be lost. So we do have to assume _something_.

Since they (fastapi and starlette) currently do not initialize logging themselves, not even inadvertently (e.g. by calling `logging.debug` instead of `logger.debug` anywhere), as any well behaved library should be expected to, I guess your change _is_ actually preferable: they get to set default levels for their loggers, then we get to override whatever we want plus setup our handlers and formatters.